student/tests/views/test_registration_views.py

```diff
@@ -90,30 +73,32 @@ def test_registration_form_renders_batch_on_start_date(batch_factory):
     assert 'id="id_batch_selection-batch_0"' in str(response.content)
 
 
-def test_registration_wizard_form_existing_user(batch_factory, existing_user):
+def test_registration_wizard_form_existing_user(batch_factory, user_factory):
     batch = batch_factory()
+    existing_user = user_factory()
```

Comment:
Nice refactor

Reply:
haha i realise it doesn't work the way i want, so need to take another look